code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -165,10 +165,13 @@
 					var/targetitem = input("Select item to search for.", "Item Mode Select","") as null|anything in item_names
 					if(!targetitem)
 						return
-					var/list/obj/item/target_candidates = get_all_of_type(item_paths[targetitem], subtypes = TRUE)
-						for(var/obj/item/candidate in target_candidates)
-							if(!is_admin_level(candidate.loc.z))
-								target = candidate
+					
+					var/list/target_candidates = get_all_of_type(item_paths[targetitem], subtypes = TRUE)
+					for(var/obj/item/candidate in target_candidates)
+						if(!is_admin_level(candidate.loc.z))

```

Comment:
shouldn't this use get_turf anyways instead?

Reply:
get_turf wouldn't give you the z level

Reply:
Damn you just found another bug that caused documents and the like not to show up on the pinpointer

Reply:
Update. Doesn't seem to have fixed that